ledger-service/http-json-cli/src/main/scala/com/daml/http/Config.scala

```diff
@@ -207,9 +208,8 @@ private[http] object WebsocketConfig extends ConfigCompanion[WebsocketConfig]("W
   )
 
   override def create(
-      x: Map[String, String],
-      supportedJdbcDriverNames: Set[String],
-  ): Either[String, WebsocketConfig] =
+      x: Map[String, String]
+  )(implicit readCtx: DummyImplicit): Either[String, WebsocketConfig] =
```

Comment:
Why is this necessary? Can we get away with renaming `create` to something else instead of introducing a dummy implicit?

Reply:
There has to be something there, to match the abstract signature.

My first instinct was initially to leave the argument where it was, but passing `()` explicitly was a little inelegant, but more importantly, it wouldn't mesh as well with supplying the `Read` instances.